### DISCUSSION TITLE
Bankwire email - make it easier for the customer

### DISCUSSION THREAD

**Discussion by LarsGMortensen** [PrestaShop/PrestaShop#33659] [Ideas]
### The problem

As a customer, when I choose to pay by bankwire, I the the "bankwire"-email.
In that email there is a list of information that I need to use for making the banktransfer (I only have the Danish language installed, sorry):

![image](https://github.com/PrestaShop/PrestaShop/assets/26773745/76552d25-3a25-45b2-b00b-664b3973a0fc)
(Note: I have inserted the highlighted line)

Notice how the customer is kindly asked to specify the order-reference in the text on the banktransfer - but we don't supply what the order-reference is (in that list). The order-reference is supplied above in the same email, sure. But why not make it easier for the customer and avoid confusion, but also inserting it in this list of "information that the customer will need to make the payment".

### The solution

Thus, as you can see in the screenshot above, I have inserted the highlighted line, containing the order-reference to make it easier and less confusing for the customer.

Oh, and by the way: In the Danish language version of Prestashop, the line "bankoverførsels beskrivelsen." should be changed to "bankoverførslens posteringstekst." which is the correct term in Danish.

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by MatShir** [comment]
For the translation, you need to check the crowdin -> https://crowdin.com/project/prestashop-official/da
But it looks a great idea! If you feel to contribute, you can link this discussion into the PR (On the raw issue)
